rust/connlib/tunnel/src/device_channel/tun_windows.rs

```diff
@@ -121,7 +121,12 @@ impl Tun {
             .creation_flags(CREATE_NO_WINDOW)
             .arg("-Command")
             .arg(format!(
-                "Set-DnsClientServerAddress -InterfaceIndex {iface_idx} -ServerAddresses(\"{DNS_SENTINEL}\")"
+                "Set-DnsClientServerAddress -InterfaceIndex {iface_idx} -ServerAddresses({})",
+                dns_config
```

Comment:
This will have 2 elements, the IPv4 and IPv6 range, right?

Reply:
Or it has the same number of elements as upstream DNS servers, and changing upstream DNS will signal connlib to re-initialize?

Reply:
This will have a sentinel dns per upstream or system underlying dns